[WIP] Immport upload_files tool

emily101-gif opened the pull request:
Adding upload_files tool to AZ immport

emily101-gif committed:
Update immport-tools.yaml

pcm32 commented:
Thanks @emily101-gif this looks good provided that such a tool identifier for that user exists in the Galaxy toolshed (which I don't think it is yet the case - see https://toolshed.g2.bx.psu.edu/view/immport-devteam). So we should wait until that happens to merge this, as the merge will trigger the CI on jenkins to try to install that tool. Ok?

emily101-gif committed:
Adding edit_fcs_markers tool

The edit_fcs_markers tool repository was already in AstraZeneca-Omics/immport-galaxy, but missing from the AstraZeneca-Omics/flow cytometry-galaxy-tools-setup/immport-tools.yaml file.

emily101-gif commented:
Sounds good. Is the plan that we will have our own toolshed with the additional tools in immport and not in the public toolshed? In the mean time I will keep sending PRs with other tools missing if that's alright @pcm32.

pcm32 commented:
Well, ideally we will manage to get everything pushed to the toolshed with the CI and their immport-devteam user. If that doesn't work, we could recourse to using our own user instead, on the same public toolshed. We might have a private toolshed if there are tools that for some reason cannot go into the public toolshed.

emily101-gif committed:
Add tools in immport-galaxy-tools/flowtools/

- fcs_merge_downsample 
- flowsom_compare
- flowsom_cross_comp
- flowsom_tree
- flowviz_density_plots
- ggcyto_1d_density_plots
